### ISSUE TITLE
proxy and https feed

### ISSUE THREAD

**Issue by erwanm** [wking/rss2email#66]
Hi there,

I'm a happy user of rss2email, so first thank you for the great work!

Recently I had a small issue with a feed which is apparently available only as https (this one: https://theintercept.com/feed/?rss)

I thought that this might be related to the fact that I'm using rss2email behind a proxy. I have the PROXY variable set in the config and it works for all the other feeds, but I was wondering if maybe rss2email doesn't use the http proxy for an https address?

Disclaimer: my skills in network-related things being close to zero, there's a real possibility that I'm completely wrong and missed something obvious!

Best,
Erwan

**Comment by brycied00d** [comment]
In its current implementation, r2e's `proxy` directive is only to (and I quote the man page) "Set an HTTP proxy" - it does not set a proxy to be used for https:// connections.

However if you set the `https_proxy` environment variable, Python's urllib should automatically use that for your HTTPS proxy. For example you could call r2e like so: `https_proxy=https://myworkproxy:443 r2e run`

**Comment by erwanm** [comment]
@brycied00d Thank you for your advice, it helped me understand what was going on: it didn't work at first, but then I realized that maybe the fact that I have the http PROXY setting in the config file forces connection through http, hence preventing using the https proxy. So I removed it from the config file, and now everything works using only the http_proxy and https_proxy environment variables.

Thank you very much!
